app/scripts/components/common/card/index.tsx

```diff
@@ -287,10 +288,11 @@ function CardComponent(props: CardComponentPropsType) {
   } else {
     const { linkTo, onLinkClick,  } = props;
     linkProperties = {
-      to: linkTo,
+      linkTo,
       onLinkClick,
       pathAttributeKeyName: 'to',
-      linkComponent: SmartLink
+      // @ts-expect-error SmartLink needs to be lazily loaded temporarily to prevent ui-library from loading react-router-dom
+      LinkElement: SmartLink
```

Comment:
Relooked at your [previous PR](https://github.com/NASA-IMPACT/veda-ui/pull/1159/files#diff-f936775422e987f9214c5bbdc155e139c0404494103a14f1da0792e2825beef3) - so looks like we have to default to SmartLink here because GHG is unable to pass this element into the card? We may just want to update this on GHG side to pass the LinkElement in instead of this non-ideal temporary solution? Unsure if thats too much of a lift right now though so this comment is non-blocking. But could we leave a `@TODO` or `@NOTE` too here with the same note you have?

Reply:
Yeah, it will be best to just change the code from the instance side. But I think it is easy to miss it with R3 and everything, so put the temporary measure here. I left a note.